### PR TITLE
[IT-1490] Feature/ec2 maintenance

### PR DESCRIPTION
**Pull request by jonryser**
Instances (there's only one running at the moment) in the iAtlasProdAccount are
missing updates because there is no maintenance window scheduled to run in
the account.  This adds the maintenance schedule and tags

### CODE REVIEW

**Review by zaro0508 — CHANGES_REQUESTED** [review]
Can you please explain the purpose of this PR in the PR description?  Why do we need it?

**Comment by ahayden** [comment]
re. IT-1490

File: config/common/maintenance.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+template:
+  type: http
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.2/maintenance.yaml
```
Comment (zaro0508) [inline config/common/maintenance.yaml:3]: latest version is [v0.3.6](https://github.com/Sage-Bionetworks/aws-infra/tags) so probably should go with that.

**Review by zaro0508 — APPROVED** [review]
thanks @ahayden, issue or not, it is always helpful to provide a nice description.  

The PR seems fine although since we are using ECS I think it would be better to change from ECS+EC2 to ECS+Fargate,  https://opsani.com/resources/aws-fargate-what-are-the-positives-and-negatives   Fargate manages a bunch of stuff for us including keeping instances patched.

**Review by andrewelamb — APPROVED** [review]

**Pull request merged**